exercises/pangram/description.md

```diff
@@ -2,5 +2,5 @@ Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan
 "every letter") is a sentence using every letter of the alphabet at least once.
 The best known English pangram is "The quick brown fox jumps over the lazy dog."
 
-The alphabet used is ASCII, and case insensitive, from 'a' to 'z'
-inclusively.
+The alphabet used consists of ASCII letters `a` to `z`, inclusive, and is case
+insensitive. Input should not contain non-ASCII symbols.
```

Comment:
This sentence seems a bit misplaced or misworded, since I think of "should" as either

1. Stating something that is most likely the case, but speaker is not sure
2. Stating a requirement that the reader is expected to conform to

 Were I a student reading this in the README that was given to me via `exercism fetch`, I would think "Should? Didn't you just serve me this exercise and its test cases? You know for sure whether the input contains non-ASCII symbols! Or are you asking me to write my own test cases?"

The sentence seems directed at, say, a contributor to problem-specifications, rather than a student. Does that sound accurate?